src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/InitializeIndexProperties.sql

```diff
@@ -0,0 +1,29 @@
+﻿GO
+CREATE PROCEDURE [dbo].[InitializeIndexProperties]
```

Comment:
>[dbo].[InitializeIndexProperties] [](http://example.com/codeflow?start=17&length=33)

Remove []

Reply:
Ok, I have removed the [] in next commit.